src/LiveSplit.Subsplits/UI/Components/SplitComponent.cs

```diff
@@ -1169,9 +1205,11 @@ protected float CalculateLabelsWidth()
             int mixedCount = ColumnsList.Count(x => x.Type is ColumnType.DeltaorSplitTime or ColumnType.SegmentDeltaorSegmentTime);
             int deltaCount = ColumnsList.Count(x => x.Type is ColumnType.Delta or ColumnType.SegmentDelta);
             int timeCount = ColumnsList.Count(x => x.Type is ColumnType.SplitTime or ColumnType.SegmentTime);
+            int varCount = ColumnsList.Count(x => x.Type is ColumnType.CustomVariable);
             return (mixedCount * (Math.Max(MeasureDeltaLabel.ActualWidth, MeasureTimeLabel.ActualWidth) + 5))
                 + (deltaCount * (MeasureDeltaLabel.ActualWidth + 5))
-                + (timeCount * (MeasureTimeLabel.ActualWidth + 5));
+                + (timeCount * (MeasureTimeLabel.ActualWidth + 5))
+                + (varCount * (Math.Max(MeasureDeltaLabel.ActualWidth, MeasureTimeLabel.ActualWidth) + 5));
```

Comment:
Why is that the width of the column, if it has nothing to do with deltas or times?